src/assets/js/gfpdf-backbone.js

```diff
@@ -1068,7 +892,8 @@
         /**
          * Load our settings dependancy
          */
-        new help.ContainerView()
+
+        // new help.ContainerView() - removed and moved into React
```

Comment:
Because we use Git for version control, there's no need to comment out code like this. Just remove.

Reply:
Fixed